frontend/src/components/DatePicker/components/DatePickerInput.tsx

```diff
@@ -2,7 +2,7 @@ import { useMemo } from 'react'
 import ReactInputMask from 'react-input-mask'
 import {
   forwardRef,
-  Input,
+  Input as ChakraInput,
```

Comment:
curious, why is this renamed to `ChakraInput`

Reply:
Spoke to tejas, added to be consistent with rest of code where when ChakraInput is used, it is renamed with alias. So this change makes sense 👍